packages/alchemy/src/config/types.ts

```diff
@@ -49,6 +51,14 @@ export type CreateConfigProps = ConnectionConfig & {
    * than for your Bundler or Node RPC calls.
    */
   signerConnection?: ConnectionConfig;
+  /**
+   * Enable this parameter if you are using the config in an SSR setting (eg. NextJS)
+   * Turing this setting on will disable automatic hydration of the client store
+   */
+  ssr?: boolean;
+
+  // TODO: should probably abstract this out into a function
+  storage?: (config?: { sessionLength: number }) => Storage;
```

Comment:
ugh forgot about this... UGHGHG

sessionLength is also not really valid for all storage types either... so I need to create a storage abstraction for this... I'm gonna... comeback to this

Reply:
I'm gonna do this later I think